src/utils/name-utils.ts

```diff
@@ -0,0 +1,13 @@
+import { get } from 'lodash';
+import { BoundaryLayerProps } from '../config/types';
+
+export function getLocationName(
```

Comment:
I like `getFullName` better than `getLocationName` thoughts?

Reply:
changed to `getFullLocationName`